web/frontend/src/components/pcd/pcd-view.vue

```diff
@@ -259,13 +259,14 @@ const update = (cloud: Uint8Array) => {
     (mesh.material as THREE.MeshBasicMaterial).dispose();
   }
 
-  const points = loader.parse(cloud.buffer, '');
+  const points = loader.parse(cloud.buffer);
```

Comment:
all new type errors here

Reply:
@micheal-parks @nfranczak following up on this, I see a type error with this line. According to the types defined in `PCDLoader.d.ts` the `loader.parse` function expects a `url` as a second argument.

Reply:
i can fix this in the PR for https://viam.atlassian.net/browse/RSDK-2315
i will also tag both of you for review when it comes out to ensure it was handled correctly.